pkg/api/controller.go

```diff
@@ -167,7 +167,7 @@ func (c *Controller) LinkPhysicalAddress(w http.ResponseWriter, r *http.Request,
 	entry := catalog.DBEntry{
 		CommonLevel:     false,
 		Path:            params.Path,
-		PhysicalAddress: params.Path,
+		PhysicalAddress: *body.Staging.PhysicalAddress,
```

Comment:
my bad :( - thanks!

Reply:
A wise singer once told me,

![If you liked it then you should have put a test on it](https://user-images.githubusercontent.com/7802932/113508321-05173600-9558-11eb-9868-40e0eedc700c.png).  (Doing that now...)